product-service/src/db.ts

```diff
@@ -0,0 +1,38 @@
+import * as productsList from './mock/products.json';
+
+export interface Product {
+    id: string;
+    imageURL: string;
+    price: string;
+}
+
+class DatabaseServiceMock {
+    products: Map<string, Product>;
+
+    constructor() {
+        this.products = new Map<string, Product>();
+        Array.from(productsList.keys()).forEach(productKey => {
+            const productItem = productsList[productKey];
+            this.products.set(productItem.id, {
+                id: productItem.id,
+                imageURL: productItem.imageURL,
+                price: productItem.price,
+            });
+        });
+    }
+
+    async getProducts() {
```

Comment:
Why async (without await)? Just use Promise.resolve() or create response handler function

Reply:
Removed `async` keyword from DB service and changed return value with `Promise.resolved` wrapped.